### PR TITLE
docs:imporve hook documentation

### PR DESCRIPTION
**Pull request by hiahia12**
#### What type of PR is this?
docs
#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) Translate the PR title into Chinese.

完善hook文档
#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: content/en/docs/hertz/tutorials/basic-feature/hooks.md [hunk 1]
```diff
@@ -171,3 +171,95 @@ hertz.go:77: [Info] HERTZ: Begin graceful shutdown, wait at most num=2 seconds..
...
+        hlog.Info("pong")
+        ctx.JSON(consts.StatusOK, utils.H{"ping": "pong"})
+    })
+
+    h.Spin()
+
+}
+
+```
+Note:After sending the request,`OnAccept` function will be printed in the terminal.
```
Comment (rogerogers) [inline content/en/docs/hertz/tutorials/basic-feature/hooks.md:215]: 大代码块一般上下单独留一行

File: content/en/docs/hertz/tutorials/basic-feature/hooks.md [hunk 2]
```diff
@@ -171,3 +171,95 @@ hertz.go:77: [Info] HERTZ: Begin graceful shutdown, wait at most num=2 seconds..
...
+```
+main.go:32: [Info] run the onAccept
+main.go:38: [Info] pong
+```
+
+
+## OnConnect
+`OnConnect` is a function to be called **after** adding it to epoll.The difference is that onConnect can get data but OnAccept cannot.
```
Comment (rogerogers) [inline content/en/docs/hertz/tutorials/basic-feature/hooks.md:223]: 标点后却个空格

File: content/zh/docs/hertz/tutorials/basic-feature/hooks.md [hunk 3]
```diff
@@ -173,3 +173,92 @@ hertz.go:77: [Info] HERTZ: Begin graceful shutdown, wait at most num=2 seconds..
...
+}
+
+```
+提示:在发出请求后,将在控制台打印`OnAccept`函数的日志。
+```
+main.go:32: [Info] run the onAccept
+main.go:38: [Info] pong
+```
+## OnConnect
+`OnConnect` 是一个在其被添加到epoll后调用的函数.它和OnAccept的不同之处在于它可以获取数据但是OnAccept不可以。
```
Comment (rogerogers) [inline content/zh/docs/hertz/tutorials/basic-feature/hooks.md:222]:
```suggestion
`OnConnect` 是一个在其被添加到 epoll 后调用的函数。它和 OnAccept 的不同之处在于它可以获取数据但是 OnAccept 不可以。
```

File: content/en/docs/hertz/tutorials/basic-feature/hooks.md [hunk 4]
```diff
@@ -171,3 +171,96 @@ hertz.go:77: [Info] HERTZ: Begin graceful shutdown, wait at most num=2 seconds..
...
+Note: After sending the request, `OnAccept` function will be printed in the terminal.
+```
+main.go:32: [Info] run the onAccept
+main.go:38: [Info] pong
+```
+
+
+## OnConnect
+`OnConnect` is a function to be called **after** adding it to epoll. The difference is that onConnect can get data but OnAccept cannot.
+```shell
```
Comment (rogerogers) [inline content/en/docs/hertz/tutorials/basic-feature/hooks.md:225]: 这是go代码

File: content/en/docs/hertz/tutorials/basic-feature/hooks.md [hunk 5]
```diff
@@ -171,3 +171,96 @@ hertz.go:77: [Info] HERTZ: Begin graceful shutdown, wait at most num=2 seconds..
 main.go:17: [Info] run shutdown hook
 engine.go:276: [Info] HERTZ: Execute OnShutdownHooks timeout: error=context deadline exceeded
 ```
+## OnAccept
+`OnAccept` is a function to be called **after** connection accepted
+but **before** adding it to epoll.
+
+```shell
```
Comment (rogerogers) [inline content/en/docs/hertz/tutorials/basic-feature/hooks.md:178]: go

File: content/zh/docs/hertz/tutorials/basic-feature/hooks.md [hunk 6]
```diff
@@ -173,3 +173,92 @@ hertz.go:77: [Info] HERTZ: Begin graceful shutdown, wait at most num=2 seconds..
 main.go:17: [Info] run shutdown hook
 engine.go:276: [Info] HERTZ: Execute OnShutdownHooks timeout: error=context deadline exceeded
 ```
+## OnAccept
+`OnAccept` 是一个在连接建立后且被添加到epoll前调用的函数。
+
+```shell
```
Comment (rogerogers) [inline content/zh/docs/hertz/tutorials/basic-feature/hooks.md:179]:
```suggestion
```go
```

File: content/zh/docs/hertz/tutorials/basic-feature/hooks.md [hunk 7]
```diff
@@ -173,3 +173,92 @@ hertz.go:77: [Info] HERTZ: Begin graceful shutdown, wait at most num=2 seconds..
...
+
+```
+提示：在发出请求后，将在控制台打印 `OnAccept` 函数的日志。
+```
+main.go:32: [Info] run the onAccept
+main.go:38: [Info] pong
+```
+## OnConnect
+`OnConnect` 是一个在其被添加到 epoll 后调用的函数。它和 OnAccept 的不同之处在于它可以获取数据但是 OnAccept 不可以。
+```shell
```
Comment (rogerogers) [inline content/zh/docs/hertz/tutorials/basic-feature/hooks.md:223]:
```suggestion
```go
```

File: content/zh/docs/hertz/tutorials/basic-feature/hooks.md [hunk 8]
```diff
@@ -173,3 +173,92 @@ hertz.go:77: [Info] HERTZ: Begin graceful shutdown, wait at most num=2 seconds..
...
+}
+
+```
+提示：在发出请求后，将在控制台打印 `OnAccept` 函数的日志。
+```
+main.go:32: [Info] run the onAccept
+main.go:38: [Info] pong
+```
+## OnConnect
+`OnConnect` 是一个在其被添加到 epoll 后调用的函数。它和 OnAccept 的不同之处在于它可以获取数据但是 OnAccept 不可以。
```
Comment (rogerogers) [inline content/zh/docs/hertz/tutorials/basic-feature/hooks.md:222]:
```suggestion
`OnConnect` 是一个在其被添加到 epoll 后调用的函数。它和 `OnAccept` 的不同之处在于它可以获取数据但是 `OnAccept` 不可以。
```

**Comment by hiahia12** [comment]
@rogerogers 
抱歉在这个地方出现了差错...刚刚已经修改好了

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**